apps/explorer/src/pages/validator/ValidatorStats.tsx

```diff
@@ -0,0 +1,227 @@
+                                    color="steel-darker"
+                                >
+                                    --
+                                </Heading>
+                            </Stats>
+                            <Stats
+                                label="Delegated Staked"
+                                tooltip="Coming soon"
+                            >
+                                <Heading
+                                    as="h3"
+                                    variant="heading3/semibold"
+                                    color="steel-darker"
+                                >
+                                    {validator.delegatedStakePercentage}%
+                                </Heading>
+                            </Stats>
+                            <Stats label="Self Staked" tooltip="Coming soon">
+                                <Heading
+                                    as="h3"
+                                    variant="heading3/semibold"
+                                    color="steel-darker"
+                                >
+                                    --
+                                </Heading>
+                            </Stats>
+                        </div>
+                    </div>
+                </Card>
+            </div>
+            <div className="basis-full md:basis-1/4">
+                <Card spacing="lg">
+                    <div className="flex  max-w-full flex-col flex-nowrap gap-8">
+                        <Heading
+                            as="div"
+                            variant="heading4/semibold"
+                            color="steel-darker"
+                        >
+                            Validator Staking Rewards
+                        </Heading>
+                        <div className="flex flex-col flex-nowrap gap-8">
+                            <Stats label="Last Epoch" tooltip="Coming soon">
+                                <Heading
+                                    as="h3"
+                                    variant="heading3/semibold"
+                                    color="steel-darker"
+                                >
+                                    --
+                                </Heading>
+                            </Stats>
+                            <Stats label="Total Reward" tooltip="Coming soon">
+                                <Heading
+                                    as="h3"
+                                    variant="heading3/semibold"
+                                    color="steel-darker"
+                                >
+                                    --
+                                </Heading>
+                            </Stats>
+                        </div>
+                    </div>
+                </Card>
+            </div>
+            <div className="max-w-[432px] basis-full md:basis-1/3">
```

Comment:
Why the px max width? Seems like this could easily just be done with variable flex amounts or grid?